src/types.ts

```diff
@@ -5,6 +5,7 @@ export type SEOConfig = {
   globals?: string[]
   uploadsCollection?: string
   fields?: Partial<Field>[]
+  tabbedUi?: boolean
```

Comment:
Change the casing of this to `tabbedUI`, then we're golden.

Reply:
Done 👍